### PR TITLE
Fixup test environment

### PR DESCRIPTION
**Pull request by kotfic**
This resolves some issues with running the test environment locally in both python 2.7 and python 3.4. It adds a docker-compose.yml file for bringing up postgis and memcached which are required for the tests to execute successfully.  

To test (from a fresh checkout of this branch):
```sh
cd tests
docker-compose up
```
in a new terminal (from the same checkout of this branch):
```sh
tox
```

For more detailed instructions see the [README](https://github.com/OpenGeoscience/ktile/blob/fixup-test-environment/tests/README.md)

### CODE REVIEW

**Comment by kotfic** [comment]
@aashish24 PTAL

File: prerequirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 gdal==1.10.0
+protobuf==3.1.0
```
Comment (aashish24) [inline prerequirements.txt:2]: @kotfic wondering why do we need protobuf perhaps I am missing something.

Comment (kotfic) [reply prerequirements.txt:2]: Protobuf is a requirement of the mapbox-vector-tiles package which is a
dependency of TileStache. The issue is the newest version of protobuf
breaks backwards compatibility,  this statement forces a downgrade of
protobuf to a version that is compatible with the current pinned
version.

See: https://github.com/OpenGeoscience/ktile/issues/20 and the commit message for https://github.com/OpenGeoscience/ktile/pull/21/commits/b5583ce10925d0057973e3673ba4d0f9fe1fa735

/Chris

Comment (aashish24) [reply prerequirements.txt:2]: Thanks for the explanation @kotfic I am assuming that before it was getting protobuf somehow (the newest one). Nonetheless, the changes looks good to me.

**Review by aashish24 — APPROVED** [review]

**Pull request merged**